src/riak_kv_entropy_manager.erl

```diff
@@ -45,6 +45,7 @@
          get_version/0,
          get_partition_version/1,
          get_pending_version/0,
+         get_upgraded/0,
```

Comment:
This is called remotely as well from other nodes when we're checking if an upgrade has already started in the cluster.